gui_fullscreen must be set before creating the window.

	-- xtraeme
--- ui/sdl2.c.orig	2014-08-01 16:12:17.000000000 +0200
+++ ui/sdl2.c	2014-09-03 12:09:44.651872098 +0200
@@ -867,6 +867,11 @@ void sdl_display_init(DisplayState *ds,
         exit(1);
     }
 
+    if (full_screen) {
+        gui_fullscreen = 1;
+        sdl_grab_start(0);
+    }
+
     for (i = 0;; i++) {
         QemuConsole *con = qemu_console_lookup_by_index(i);
         if (!con) {
@@ -898,11 +903,6 @@ void sdl_display_init(DisplayState *ds,
         g_free(filename);
     }
 
-    if (full_screen) {
-        gui_fullscreen = 1;
-        sdl_grab_start(0);
-    }
-
     mouse_mode_notifier.notify = sdl_mouse_mode_change;
     qemu_add_mouse_mode_change_notifier(&mouse_mode_notifier);
 
